playbooks/common/openshift-cluster/openshift-hosted/openshift_logging.yml

```diff
@@ -0,0 +1,35 @@
+---
+- name: Create Hosted Resources
+  hosts: oo_first_master
+  pre_tasks:
+  - set_fact:
+      logging_hostname: "{{ openshift_hosted_logging_hostname | default('kibana.' ~ (openshift_master_default_subdomain | default('router.default.svc.cluster.local', true))) }}"
+      logging_ops_hostname: "{{ openshift_hosted_logging_ops_hostname | default('kibana-ops.' ~ (openshift_master_default_subdomain | default('router.default.svc.cluster.local', true))) }}"
+      logging_master_public_url: "{{ openshift_hosted_logging_master_public_url | default(openshift_master_public_api_url) | default(None) }}"
+      logging_elasticsearch_cluster_size: "{{ openshift_hosted_logging_elasticsearch_cluster_size | default(1) }}"
+      logging_elasticsearch_ops_cluster_size: "{{ openshift_hosted_logging_elasticsearch_ops_cluster_size | default(1) }}"
+  roles:
+  - role: openshift_logging
```

Comment:
A lot of the following variables could probably move to the role defaults for logging.  This is beyond the scope of this PR.  @ewolinetz

Reply:
Agree and agree.